src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.card__stars {
```

Remove redundant mix `card__stars`
```suggestion
stars {
```